test/_log.js

```diff
@@ -527,4 +528,15 @@ describe('Debug Logging Utility', () => {
     });
   });
 });
-
+describe('Testing manual update', () => {
+  it('Should return a string', () => {
+    v.update.updateManual('1.0.0', 'test');
+    expect('1.0.0');
+  });
+});
+describe('Testing Auto update', () => {
+  it('Should return a string', () => {
+    v.update.updateAuto('test');
+    expect('package.json updated');
```

Comment:
Us a sinon spy here and rewire to prevent an actual change from happening to your package.json when testing.